22-discovery.ks

```diff
@@ -22,6 +22,11 @@ cat > /etc/udev/rules.d/81-nm-prepare.rules <<'UDEV'
 ACTION=="add", SUBSYSTEM=="net", NAME!="lo", RUN+="/usr/bin/systemd-cat -t nm-prepare /usr/bin/nm-prepare %k"
 UDEV
 
+echo " * disable IPv6"
```

Comment:
Why would you want to do this?

Reply:
We already do this but per individual connections, I want this globally. Until we decide to support IPv6 in discovery.

Reply:
You know what - removed from this patch, it's irrelevant here. But please note IPv6 is disabled on connection level atm.